src/main/java/com/app/notebook/model/File.java

```diff
@@ -0,0 +1,59 @@
+package com.app.notebook.model;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * notebook-api Created by Catalin on 10/21/2020
+ */
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "files")
+public class File {
+
+    @Id
+    @GeneratedValue(generator = "uuid2")
+    @GenericGenerator(name = "uuid2", strategy = "org.hibernate.id.UUIDGenerator")
```

Comment:
delete this shit from everywhere

Reply:
astea o sa fie schimbate cand o sa fie postgres, h2 nu stie out of the box cum sa genereze

Reply:
daca ai liquibase facut.. nu mai ai nevoie de astea aici :P

Reply:
punem liquibase sa fii impacat sufleteste :))